### PR TITLE
Leaderboard_page

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

**Review by val-fcn — DISMISSED** [review]

**Review by val-fcn — DISMISSED** [review]

**Review by Adwarf14 — DISMISSED** [review]

File: frontend/src/components/LeaderBoard.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./LeaderBoard.scss";
+
+function LeaderBoard() {
+  return (
+    <div className="leaderBoard">
```
Comment (LoicBrassart) [inline frontend/src/components/LeaderBoard.jsx:6]: I think it'd be better to use *one* `<ol>` tag and to split the data in two in it (ex: flexbox)
The idea is, it'll be easier to fill these data when you use a real-world API

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A little structure problem in this PR, but nothing too harsh I think ;-)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some things to change

**Review by Adwarf14 — DISMISSED** [review]

**Review by BlandineM — APPROVED** [review]

File: frontend/src/components/LeaderBoard.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      padding: 10px;
+      align-self: center;
+    }
+  }
+
+  .leaderBoardItems {
+    width: 30%;
+    ol {
+      width: 100%;
+      counter-reset: my-awesome-counter;
```
Comment (LoicBrassart) [inline frontend/src/components/LeaderBoard.scss:34]: ... Wat ?

**Review by LoicBrassart — COMMENTED** [review]
There's something I didn't understand, please explain it to me

**Review by LoicBrassart — APPROVED** [review]
Ok!

**Pull request merged**